nomenclature/__init__.py

```diff
@@ -8,6 +11,11 @@
     RegionAggregationMapping,
 )
 
-
 # set up logging
 logger = logging.getLogger(__name__)
+
+# get version number either from git (preferred) or metadata
+try:
+    __version__ = get_version(Path(__file__).parent.parent)
```

Comment:
Pathlib actually has a function to traverse the entire file path.
```suggestion
    __version__ = get_version(Path(__file__).parents[1])
```
Should be the correct one.